docs/referenceConf.js

```diff
@@ -206,7 +210,7 @@ exports.config = {
     //
     // If you need access back to the current configuration object,
     // use a pattern like the following:
-    //     browser.getProcessedConfig().then(function(config) {
+    //     return browser.getProcessedConfig().then(function(config) {
```

Comment:
Ok, that's fine. I only mentioned that because there are multiple ways of using this information without returning it, but you could be right that returning the value serves as an important use case.